data-ingestion-apis/opentelemetry-metrics.mdx

```diff
@@ -1,42 +1,77 @@
 ---
-title: "Data Ingestions API"
+title: "Customizing Telemetry Ingestion"
+sidebarTitle: "OpenTelemetry"
 ---
 
 # Overview
 This guide provides an overview and setup instructions for Middleware’s OpenTelemetry Data Ingestion API. The OpenTelemetry Ingestion API has two endpoints: [metrics](#metrics-endpoint) and [logs](#logs-endpoint).
 
-## Resource Types
 
-For both endpoints, the resource type attribute groups the ingested data under the specified label on Middleware dashboards and reports. The resource type field requires one of the following options.
+# Prerequisites
 
+1. Middleware Agent needs to already be installed
+2. App must already be configured to collect telemetry 
 
-| **Group** | **Resource Type** |
-| :---------: | :---------------: |
-| host | host.id |
-| k8s.node | k8s.node.uid |
-|k8s.pod | k8s.pod.uid |
-| k8s.deployment | k8s.deployment.uid |
-| k8s.daemonset | k8s.daemonset.uid |
-| k8s.replicaset| k8s.replicaset.uid |
-| k8s.statefulset | k8s.statefulset.uid |
-| k8s.namespace | k8s.namespace.uid |
-| service | service.name |
-| os | os.type |
-| custom | custom |
+### Step 1: Setup Resource Attributes
```

Comment:
Agent is not required, however there is an extra config that needs to be applied if you're bypassing/not using the agent. That config is `target`, ed has details on how to apply it.